reana/reana_dev/release.py

```diff
@@ -202,4 +214,82 @@ def release_pypi(ctx, component, timeout):  # noqa: D301
         click.secho(f"{component} successfully released on PyPI", fg="green")
 
 
+@click.option(
+    "--version", "-v", help="REANA version to release, e.g. 0.7.0a1",
+)
+@click.option("--user", "-u", default="reanahub", help="DockerHub user name [reanahub]")
+@release_commands.command(name="release-helm")
+@click.pass_context
+def release_helm(ctx, version, user):  # noqa: D301
+    """Release REANA as a Helm chart.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+    """
+
+    def _update_values_yaml(new_docker_images):
+        """Update all images in ``values.yaml``, skipping the ones up to date."""
+        update_images_cmds = []  # FIXME
+        values_yaml_relative_path = "helm/reana/values.yaml"
+        values_yaml_abs_path = os.path.join(
+            get_srcdir("reana"), values_yaml_relative_path
+        )
+        values_yaml = ""
+
+        with open(values_yaml_abs_path) as f:
+            values_yaml = f.read()
+            for docker_image in new_docker_images:
+                image_name, _ = docker_image.split(":")
+                if image_name in values_yaml:
+                    values_yaml = re.sub(
+                        f"{image_name}:.*", lambda _: docker_image, values_yaml, count=1
+                    )
+
+        with open(values_yaml_abs_path, "w") as f:
+            f.write(values_yaml)
+
+    remaining_docker_releases = []
+    new_docker_images = []
+    for component in REPO_LIST_CLUSTER:
+        if not is_component_dockerised(component):
+            continue
+        if not git_is_current_version_tagged(component):
+            remaining_docker_releases.append(component)
+        else:
+            new_docker_images.append(
+                f"{user}/{component}:{get_component_version_as_semver2(component)}"
+            )
+
+    if remaining_docker_releases:
```

Comment:
@tiborsimko currently this check causes the following error:
```console
reana-dev release-helm                                                                                                              [5:34:17]
[2020-09-02T17:34:17] reana-job-controller: git describe --tags
[2020-09-02T17:34:17] reana-server: git describe --tags
[2020-09-02T17:34:18] reana-workflow-controller: git describe --tags
The following components are missing to be released:
reana-auth-vomsproxy
reana-ui
reana-message-broker
reana-workflow-engine-cwl
reana-workflow-engine-serial
reana-workflow-engine-yadage
```
Because we do not have tags for previous versions. Should we make this check optional? Or should we create tags for the previous releases?

Reply:
Hmm, we do have tag `0.7.0a1` for `r-w-e-serial` for example?